inventories/seapath_cluster_definition_example.yml

```diff
@@ -136,16 +136,9 @@ all:
                         #OPTIONAL
                         logstash_server_ip: 10.10.10.10
 
-                        # Realtime/CPUs cgroups isolation configuration
-                        # All those parameters are optional.
-                        isolcpus: "2-5,14-17" # CPUs to isolate
-
-                        cpusystem: "0,12" # CPUs reserves for system
```

Comment:
Why not keep this variable in the example?
To me, the idea of these inventories is to give a usage example of all the possible variables. Do you want to change this?

Reply:
I agree that these inventories must provide usage examples.
However, they are also a starting point for people that want to implement SEAPATH : They copy this inventory and modify it line by line.
I think it is a good idea to keep this inventory simple for those people.

Particularly, these cpu variables are advanced variables and should not be proposed to beginners.
I think the best would be to have different kind of inventories (basic, networking, cgroups, etc ...) for examples, but I will not do that now.

Reply:
We have the `seapath_demo_cluster_definition_example.yml` inventory for beginners.